package.json

```diff
@@ -56,7 +57,7 @@
     "tslint-config-prettier": "^1.18.0",
     "tslint-config-standard": "^9.0.0",
     "tslint-eslint-rules": "^5.4.0",
-    "typescript": "^3.9.2",
+    "typescript": "^3.9.5",
```

Comment:
I needed to upgrade all the dependencies because my actual linting PR was failing CI because ESLint v6 doesn't know about `/* eslint-disable -- directive comments */`